stable/index.yaml

```diff
@@ -1,9 +1,40 @@
 apiVersion: v1
 entries:
   portworx:
+  - apiVersion: v1
+    appVersion: 2.9.1
```

Comment:
let's add the 2.10.0 after this. We should match the latest central version 
You can use [this](https://install.portworx.com/2.10?mc=false&kbver=1.20.0&b=true&kd=type%3DPremium_LRS%2Csize%3D150&s=%22type%3DPremium_LRS%2Csize%3D50%22&c=px-cluster-19d04198-1185-4fce-aea7-adf3adef6f05&aks=true&stork=true&csi=true&mon=true&tel=false&st=k8s) spec to check the latest versions